### PR TITLE
feat: Add make targets for asset upload

### PR DESCRIPTION
**Pull request by lindnerby**
**Description**

Changes proposed in this pull request:

- add usage of kyma cli to create module and upload it
- fix module-template name and usage of registry

**Related issue(s)**
* https://github.com/kyma-project/template-operator/issues/52

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -142,6 +157,52 @@ lint: ## Download & Build & Run golangci-lint against code.
 	GOBIN=$(LOCALBIN) go install github.com/golangci/golangci-lint/cmd/golangci-lint@$(GOLANG_CI_LINT_VERSION)
 	$(LOCALBIN)/golangci-lint run
 
+.PHONY: configure-git-origin
+configure-git-origin:
+	@git remote | grep '^origin$$' -q || \
+		git remote add origin https://github.com/kyma-project/eventing-manager
```
Comment (Tomasz-Smelcerz-SAP) [inline Makefile:163]: why `eventing-manager`?

File: Makefile [hunk 2]
```diff
@@ -142,6 +157,52 @@ lint: ## Download & Build & Run golangci-lint against code.
...
+
+#.PHONY: module-config-template
```
Comment (Tomasz-Smelcerz-SAP) [inline Makefile:165]: Please remove the commented lines.

File: Makefile [hunk 3]
```diff
@@ -142,6 +157,52 @@ lint: ## Download & Build & Run golangci-lint against code.
...
+#		| sed -e 's/{{.Channel}}/${MODULE_CHANNEL}/g' \
+#			-e 's/{{.Version}}/$(MODULE_VERSION)/g' \
+#			-e 's/{{.Name}}/kyma-project.io\/module\/$(MODULE_NAME)/g' \
+#				> module-config.yaml
+
 .PHONY: build-manifests
 build-manifests: manifests kustomize ## Install CRDs into the K8s cluster specified in ~/.kube/config.
 	$(KUSTOMIZE) build config/default > template-operator.yaml
+
+DEFAULT_CR ?= $(shell pwd)/config/samples/default.yaml
```
Comment (Tomasz-Smelcerz-SAP) [inline Makefile:177]: I think the path is: `config/samples/default-sample-cr.yaml`

File: Makefile (hunk 1 above)
Comment (lindnerby) [reply Makefile]: Wow, oversaw this. It's copied from eventing-manager, left-over. Fixed

File: Makefile (hunk 2 above)
Comment (lindnerby) [reply Makefile:165]: Thanks!

**Review by Tomasz-Smelcerz-SAP — APPROVED** [review]

**Pull request merged**